frontend/src/js/components/tenants/expanded-tenant.tsx

```diff
@@ -11,18 +11,197 @@
 //    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
-import { Drawer } from '@mui/material';
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { Close as CloseIcon, Link as LinkIcon } from '@mui/icons-material';
+import { Button, Checkbox, Divider, Drawer, FormControlLabel, FormHelperText, IconButton, TextField } from '@mui/material';
+import { makeStyles } from 'tss-react/mui';
+
+import actions from '@northern.tech/store/actions';
+import { getOrganization } from '@northern.tech/store/organizationSlice/selectors';
+import { editTenantDeviceLimit, removeTenant } from '@northern.tech/store/organizationSlice/thunks';
+import { AppDispatch } from '@northern.tech/store/store';
+import copy from 'copy-to-clipboard';
+
+import { generateTenantPathByName } from '../../utils/locationutils';
+import { ConfirmModal } from '../common/ConfirmModal/ConfirmModal';
+import { TwoColumns } from '../common/configurationobject';
+import { DeviceCount } from '../header/devicecount';
+import { HELPTOOLTIPS, MenderHelpTooltip } from '../helptips/helptooltips';
+import { Tenant } from './types';
 
 interface ExpandedTenantProps {
-  tenantId: string;
   onCloseClick: () => void;
+  tenant: Tenant;
 }
 
+const useStyles = makeStyles()(theme => ({
+  deleteModal: {
+    maxWidth: '380px'
+  },
+  devLimitLabel: {
+    color: theme.palette.primary.main,
+    fontSize: '11px',
+    fontWeight: 400
+  },
+  tenantTitle: {
+    fontSize: '17px',
+    fontWeight: 500
+  },
```

Comment:
Please align this with the other details drawers (so likely an h3 or at least a bold element) to keep the consistency.